db-server/test/backend/db_tests.js

```diff
@@ -2284,7 +2285,8 @@ module.exports = function (config, DB) {
         return db.getRecoveryKey(options)
           .then((res) => {
             assert.equal(res.recoveryData, data.recoveryData, 'recovery data set')
-            assert.equal(res.recoveryKeyId.toString('hex'), data.recoveryKeyId.toString('hex'), 'recovery data set')
+            const recoveryKeyId = util.createHash(data.recoveryKeyId)
```

Comment:
It's super nit-picky, but I think this test would read clearer if the variable here were named `recoveryKeyIdHash` rather than `recoveryKeyId` - for a moment I interpreted the assertion below as saying "the recovery key id hash is equal to the recovery key id" which seemed very confusing :-)